docs/reference/commandline/container_prune.md

```diff
@@ -38,6 +40,28 @@ f98f9c2aa1eaf727e4ec9c0283bc7d4aa4762fbdba7f26191f26c97f64090360
 Total reclaimed space: 212 B
 ```
 
+## Filtering
+
+The filtering flag (`-f` or `--filter`) format is of "key=value". If there is more
+than one filter, then pass multiple flags (e.g., `--filter "foo=bar" --filter "bif=baz"`)
+
+The currently supported filters are:
+
+* until (`<timestamp>`) - filter images created before given timestamp
```

Comment:
Also, instead of "filter", describe what the result is; "only remove containers created before given timestamp"